examples/examples_nodejs_test.go

```diff
@@ -565,3 +567,35 @@ func getAwsSession(t *testing.T) *session.Session {
 	require.NoError(t, err)
 	return sess
 }
+
+func TestUpdateImportedLambda(t *testing.T) {
+	test := pulumitest.NewPulumiTest(t, "lambda-import-ts", 
+		opttest.LocalProviderPath("aws", filepath.Join(getCwd(t), "..", "bin")),
+	)
+
+	test.SetConfig("runtime", "nodejs18.x")
+	res := test.Up()
+	lambdaName := res.Outputs["lambda_name"]
+	lambdaRole := res.Outputs["lambda_role"]
+
+	secondStack := test.InstallStack("new_stack")
+
+	// Check that we can reimport the lambda.
+	secondStack.SetConfig("lambda_name", lambdaName.Value.(string))
+	secondStack.SetConfig("runtime", "nodejs18.x")
+	secondStack.SetConfig("lambda_role", lambdaRole.Value.(string))
+	secondStack.Up()
+
+	// Check that we can change a property on the lambda
+	secondStack.SetConfig("runtime", "nodejs16.x")
+	secondStack.Up()
```

Comment:
There's no asserts here are we sure this is a strong test?

Reply:
The framework has the asserts by default: https://github.com/pulumi/providertest/blob/dc14d64784583753791733c6ac090d972d1ec746/pulumitest/up.go#L18

I've also verified that the test fails without the patch.